### ISSUE TITLE
Rejuvenation corrupting chunks?

### ISSUE THREAD

**Issue by LemADEC** [BuiltBrokenModding/ICBM-Classic#14]
As of 1.7.10-2.15.1b33, with assorted tech and magical mods, it was observed that rejuvenation explosion corrupt the chunk: it no longer updates properly and allow duping.
This was in single player. I didn't try to reproduce it.

**Comment by DarkGuardsman** [comment]
I've noticed something similar to this when setting off explosives nearby. If you could try to duplicate it to see if it is a constant effect.

**Comment by LemADEC** [comment]
It's frequent in SMP on 1.7.10, probably due to Cauldron/MCPC. That one was a first in single player. I'll try to reproduce it later on.

**Comment by DarkGuardsman** [comment]
ya once you throw in any non-standard server .jar it gets really complicated to know if it is a bug or not. As they change the world generation code to optimize it in some areas. Where ICBM directly uses the world generation code to make the regen work.

**Comment by LemADEC** [comment]
On a new world with the same mods, the Rejuvenation no longer have any effect.
Moving away from spawn or to other biomes won't change the result. I'm not quite sure what's going on with it.
Logs are silent:
```
[02:58:06] [Client thread/INFO] [inventorytweaks/]: codechicken.nei.ContainerCreativeInv
[02:58:17] [Client thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 103, 72, 43.
[02:58:17] [Server thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 103, 72, 43.
[02:58:17] [Client thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 43.
[02:58:17] [Server thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 43.
[02:58:18] [Client thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 101, 72, 43.
[02:58:18] [Server thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 101, 72, 43.
[02:58:19] [Client thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 44.
[02:58:19] [Server thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 44.
[02:58:20] [Client thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 42.
[02:58:20] [Server thread/DEBUG] [FML/]: LemADEC placed Debilitation Explosives in: 102, 72, 42.
[02:58:32] [Client thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 102, 73, 43.
[02:58:32] [Server thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 102, 73, 43.
[02:58:48] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
[02:59:11] [Client thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 102, 73, 43.
[02:59:11] [Server thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 102, 73, 43.
[02:59:21] [Client thread/INFO] [inventorytweaks/]: net.minecraft.client.gui.GuiChat
[02:59:23] [Client thread/INFO] [inventorytweaks/]: net.minecraft.client.gui.GuiIngameMenu
[02:59:23] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
[02:59:54] [Client thread/INFO] [inventorytweaks/]: codechicken.nei.GuiExtendedCreativeInv
[02:59:54] [Client thread/INFO] [inventorytweaks/]: codechicken.nei.ContainerCreativeInv
[02:59:55] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
[02:59:59] [Client thread/DEBUG] [FML/]: LemADEC placed Nuclear Explosives in: 102, 73, 43.
[02:59:59] [Server thread/DEBUG] [FML/]: LemADEC placed Nuclear Explosives in: 102, 73, 43.
[03:00:13] [Client thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 104, 64, 42.
[03:00:13] [Server thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 104, 64, 42.
[03:00:31] [Client thread/INFO] [inventorytweaks/]: net.minecraft.client.gui.GuiChat
[03:00:40] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
[03:01:25] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
[03:01:37] [Client thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 89, 66, 244.
[03:01:37] [Server thread/DEBUG] [FML/]: LemADEC placed Rejuvenation Explosives in: 89, 66, 244.
[03:01:49] [Client thread/INFO] [inventorytweaks/]: net.minecraft.client.gui.GuiChat
[03:02:10] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save Rgen
```

**Comment by DarkGuardsman** [comment]
I worked over the code and from I can tell it was working. It however, had an odd habit of not syncing the changes to the client. Any duplication I suspect where ghost blocks and do not actually exist server side. You can confirm this yourself just to make sure.

**Comment by LemADEC** [comment]
Fixed as of ICBM-classic-1.7.10-2.15.1b45-universal
